sale_order_type/tests/test_sale_order_type.py

```diff
@@ -143,7 +143,6 @@ def create_invoice(self, partner=False, sale_type=False):
             self.env["account.move"].with_context(default_move_type="out_invoice")
         )
         inv_form.partner_id = partner or self.partner
-        inv_form.sale_type_id = sale_type or self.sale_type
```

Comment:
@rov-adhoc IMHO, you should let this flow (let the user choose the sale type in invoice form) and create another test without filling it. Thanks